src/eei.h

```diff
@@ -136,6 +136,9 @@ class EthereumInterface {
   void eeiRevertOrFinish(bool revert, uint32_t offset, uint32_t size);
 
   // Helpers methods
+  inline std::string depthToString() const {
+    return std::string("[") + std::to_string(m_msg.depth) + "]";
```

Comment:
```suggestion
    return "[" + std::to_string(m_msg.depth) + "]";
```

Reply:
I think it wasn't compiling hence the explicit `string("[")`

Reply:
Changed.